### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Confectrician**
- Remove Habmin handling.
- Fix edit links for bundles with external repos. Fixes openhab/openhab-docs#1374
- Redirect v1 addons to the v2 subdomain.

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by Confectrician** [comment]
I have cancelled the deploy preview manually to save some build minutes. 
The build succeded before and i just fixed 3 typos.

File: prepare-docs.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@
 $docs_repo_root = $docs_repo + "/blob/main"
 $docs_repo_branch = "final"
 $addons_repo_branch = "main"
+$addons_repo_branch_outdated = "master" # Fix edit links for bundles with iown repositories.
```
Comment (ghys) [inline prepare-docs.rb:14]: Why "outdated"? :)

**Comment by ghys** [comment]
👍

File: prepare-docs.rb (hunk 1 above)
Comment (Confectrician) [reply prepare-docs.rb:14]: I have switched to main (which is prbably the generic one for future repos) for the docs and addons repo to fix all other links.
I would except the other repos to switch to `main` at some time in future too, thats why i called it outdated. :)

**Pull request merged**